erroranalysis/erroranalysis/_internal/cohort_filter.py

```diff
@@ -151,6 +152,26 @@ def _filters_has_classification_outcome(self, filters):
                         return True
         return False
 
+    def _filters_has_regression_error(self, filters):
+        """Checks if regression error is specified as a filter.
+
+        :param filters: The filters.
+        :type filters: list[dict]
+        :return: True if regression filter is specified in the filters.
```

Comment:
nit: (in doc string)
```regression filter -> regression error```
is that an improvement, what do you think?

Reply:
done.